app/models/space_cw.rb

```diff
@@ -0,0 +1,6 @@
+class SpaceCw < ApplicationRecord
+  belongs_to :user
+  has_many :reservations
+  validates :name, :description, presence: true
+  validates :price, :discount, numericality: { greater_than_or_equal_to: 0 }
+end
```

Comment:
Good job implementing relationships and validations for the SpaceCw model. Bu in this case I encorage you to avoid using contraction as in `SpaceCW`, this is not recomended because I new developer will have to figure out what Cw is, and it may cause future bugs because Ruby on Rails relies a lot about how things are named.
In this article you will fin the importance to keep names descriptive: https://webcrunch.com/posts/best-practices-for-naming-models-in-ruby-on-rails

And, in the official docs you will find that different words must be camel case in case of models:
https://guides.rubyonrails.org/active_record_basics.html#naming-conventions